cfgov/regulations3k/models/pages.py

```diff
@@ -214,7 +213,6 @@ class RegulationPage(RoutablePageMixin, SecondaryNavigationJSMixin, CFGOVPage):
 
     content = StreamField([
         ('info_unit_group', organisms.InfoUnitGroup()),
-        ('full_width_text', RegulationsFullWidthText()),
```

Comment:
I was unsure if this was correct, since this is in the RegulationPage class, but `RegulationsFullWidthText` only contained a notification in the `blocks.py`, which was removed, so seemed it should be removed here. I wasn't sure how to test this page type since it doesn't show up in the list when creating a page.

Reply:
RegulationsFullWidthText [inherits from](https://github.com/cfpb/cfgov-refresh/pull/4891/files#diff-358ce821d7c89205bf820b087d01742cL100) plain FullWidthText, so it has other blocks defined and exists on other pages. If you're removing the notification block only you might be able to replace this line with

`('full_width_text', organisms.FullWidthText()),`

If it's helpful you can [use wagtail-inventory](http://localhost:8000/admin/inventory/?form-TOTAL_FORMS=3&form-INITIAL_FORMS=0&form-MIN_NUM_FORMS=0&form-MAX_NUM_FORMS=3&form-0-has=includes&form-0-block=regulations3k.blocks.RegulationsFullWidthText&form-1-has=includes&form-1-block=&form-2-has=includes&form-2-block=) to see all of the pages that currently have a RegulationsFullWidthText.

Reply:
Cool, pushed up that update in https://github.com/cfpb/cfgov-refresh/pull/4891/commits/24a652a6978c487cb8d554822c72ac9f2c67107f
Looks like that doesn't need a data migration, unless I'm looking at the wrong thing?

Reply:
It shouldn't need a data migration, since we're just throwing the data that's there away!

Reply:
Locally, the Notification block was still visible in the page editor and it was still looking for the `regulations3k/notification.html` template when viewing the page. I had to manually remove the Notification block from the editor to get the page to preview without error. Is that expected? If so, not a big deal, we'll just need to do this on the live site, as well. Seems like we could go ahead and do it, even, given the stakeholder's response about it.

Reply:
Belay that comment; seems to have been a caching issue. After stopping and restarting my server, everything's fine.